modules/identity-provider-provisioning-user-lookup-mapping.adoc

```diff
@@ -39,18 +41,21 @@ as shown in the appropriate identity provider section below.
 +
 For example, the following commands creates an Identity with identity provider `ldap_provider` and the identity provider user name `bob_s`.
 +
+[source,terminal]
 ----
 $ oc create identity ldap_provider:bob_s
```

Comment:
As said "`In LDAP identity provider, it is a little tricky (which needs specific doc)`" above, do not choose ldap_provider as example.